packages/foreman/rubygem-unf_ext/rubygem-unf_ext.spec

```diff
@@ -105,6 +105,9 @@ rm -rf gem_ext_test
 %{gem_instdir}/unf_ext.gemspec
 
 %changelog
+* Mon Apr 20 2020 Evgeni Golov 0.0.7.2-3
```

Comment:
Does this work? Going back and forth in a changelog? Shouldn't we just revert the git commit and untag the new version?

Reply:
Sure, why not. I still want the changes that came in for the spec, just not the version. So I could drop the older changelog entry, but I don't think it will give us anything.

Reply:
Fair enough.